app/src/Users.vue

```diff
@@ -68,6 +79,7 @@ export default class Users extends Vue {
   }
 
   handleAddUser (): void {
+    console.log('here')
```

Comment:
Please review your pull request twice before publishing it and requesting review - it helps to avoid adding redundant code.

Reply:
So sorry! Some of my changes somehow did not save! I'll fix it as asap